pwndbg/ida.py

```diff
@@ -159,41 +183,46 @@ def Anterior(addr):
         lines.append(r)
     return '\n'.join(lines)
 
+
 @withIDA
 def GetBreakpoints():
     for i in range(GetBptQty()):
         yield GetBptEA(i)
 
+
 @withIDA
 def GetBptQty():
     return _ida.GetBptQty()
 
+
 @withIDA
 @returns_address
 def GetBptEA(i):
     return _ida.GetBptEA(i)
 
-_breakpoints=[]
+
+_breakpoints = []
+
 
 @pwndbg.events.cont
 @pwndbg.events.stop
 @withIDA
 def UpdateBreakpoints():
     # XXX: Remove breakpoints from IDA when the user removes them.
     current = set(eval(b.location.lstrip('*')) for b in _breakpoints)
-    want    = set(GetBreakpoints())
+    want = set(GetBreakpoints())
 
     # print(want)
 
-    for addr in current-want:
+    for addr in current - want:
```

Comment:
I'm adding `setup.cfg` to the project root, which contains the `pep8.py` options.

Since `pep8.py` picks these up automagically, so should PyCharm.